php/admin_page.php

```diff
@@ -10,50 +38,61 @@
     <link rel= "stylesheet" type="text/css" href= "../admin_style.css">
 </head>
 <body>
-    <div class="mainText">
-        <div class="container">
+    <div class="mainText container">
             <h4>Welcome</h4>
             <h2>RICHARD</h2>
             <h3>'ABOUT ME'</h3>
-        </div>
     </div>
-
     <div class="forms container">
         <div class="formSize">
             <p>Add additional paragraphs</p>
-            <form method="post" action="admin_add.php" class="formAdd">
-                <textarea rows="6" cols="80" name="add_form" required></textarea>
+            <form method="post" action="admin_add.php" class="formAdd editParaRight">
+                <textarea name="add_form"  required></textarea>
+                <div>
                 <input type="submit" name="addSubmit">
+                </div>
             </form>
         </div>
         <div class="formSize">
             <p>Edit paragraph</p>
-            <form>
-                <textarea rows="6" cols="80" name="edit_form"></textarea>
+            <form method="post" class="formText" action="admin_page.php">
                 <h4>Choose which paragraph to edit</h4>
                 <select name="editSelect">
-                    <option>paragraph 1</option>
-                    <option>paragraph 2</option>
-                    <option>paragraph 3</option>
+                    <option selected="selected" value="choose">Choose Paragraph</option>
+                    <?php if(isset($editDropdownResults)){
+                        echo $editDropdownResults;
+                    } ?>
```

Comment:
These issets should be set at the top in the rest of the php I think. Not down in the html?

Reply:
apparently if it relies on $_POSTs I need to do an isset on top and one when its called

Reply:
wooops my bad, this one doesnt need an isset because its always loaded with the page